homework/index.js

```diff
@@ -29,19 +30,79 @@
     });
     return elem;
   }
+  function app(repos) {
+    const root = document.getElementById('root');
+    const container = createAndAppend('div', root, {
+      class: 'container',
+    });
+    const header = createAndAppend('header', container, {
+      class: 'header',
+    });
+    createAndAppend('p', header, {
+      text: 'HYF Repositories',
+    });
+    const select = createAndAppend('select', header, {
+      id: 'list',
+    });
+    const div = createAndAppend('div', root, { class: 'list-div' });
+    const list = createAndAppend('ul', div, { class: 'list1' });
+    const listContributor = createAndAppend('ul', div, { class: 'list2' });
+    const newList = createAndAppend('li', list, { text: ' ' });
```

Comment:
Better to use names you and other developers can remember later in time, which one do you think is better; `newList` or `descriptionList`?

Reply:
Thanks! you are right! descriptionList will be readable and understandable 👍 :)

Reply:
Good!